From 77875a5d9fb79ae7e28b6597e9bfec9a57adcc85 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 21 Mar 2021 15:48:45 +0000
Subject: [PATCH] GCC: fix type confusion in EffectPaintPropertyNode

GCC gets confused if a method and an alias have same name. This can be
avoided by adding explicit namespace.
---
 .../platform/graphics/paint/effect_paint_property_node.h        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/blink/renderer/platform/graphics/paint/effect_paint_property_node.h b/third_party/blink/renderer/platform/graphics/paint/effect_paint_property_node.h
index fa0995c..8b71d54 100644
--- a/third_party/blink/renderer/platform/graphics/paint/effect_paint_property_node.h
+++ b/third_party/blink/renderer/platform/graphics/paint/effect_paint_property_node.h
@@ -287,7 +287,7 @@ class PLATFORM_EXPORT EffectPaintPropertyNode
     return state_.compositor_element_id;
   }
 
-  const DocumentTransitionSharedElementId& DocumentTransitionSharedElementId()
+  const blink::DocumentTransitionSharedElementId& DocumentTransitionSharedElementId()
       const {
     return state_.document_transition_shared_element_id;
   }
-- 
2.26.3

